### PR TITLE
OCLOMRS-857 Back button and title hidden by expanded side nav

### PR DESCRIPTION
**Pull request by nk183**
# JIRA TICKET NAME:
[OCLOMRS-857](https://issues.openmrs.org/browse/OCLOMRS-857)

# Summary:
fix "Back button and title hidden by expanded side nav or NavDrawer"

s1

s2

### CODE REVIEW

**Comment by dkayiwa** [comment]
@nk183 can you take a look at this? https://wiki.openmrs.org/display/docs/Pull+Request+Tips

**Comment by nk183** [comment]
I already looked at [tips](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) 
I am not able to spot any problem ,please do tell me if there any problem ?

**Comment by dkayiwa** [comment]
@nk183 did you read about the JIRA workflow on the pull request tips page?

**Comment by nk183** [comment]
@dkayiwa Now I think It Okay?

File: src/components/NavDrawer.tsx [hunk 1]
```diff
@@ -80,7 +80,7 @@ const useStyles = makeStyles((theme: Theme) =>
       ...theme.mixins.toolbar,
     },
     content: {
-      padding: theme.spacing(3),
+      // padding: theme.spacing(3),
```
Comment (dkayiwa) [inline src/components/NavDrawer.tsx:83]: https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Donotcommentoutcode

**Comment by dkayiwa** [comment]
@nk183 did you also attach, on the JIRA ticket, screenshots with the effects of your changes, as advised at the pull requests page?

**Comment by nk183** [comment]
image

@dkayiwa I have already clicked on "Request Code Review" 
please tell where to add the screenshots ?

**Comment by gracepotma** [comment]
@nk183 thank you so much for your work on this project, we look forward to working with you more!
But in the meantime, please see my comment here; it seems there has become confusion between this PR and a duplicated one: https://issues.openmrs.org/browse/OCLOMRS-857?focusedCommentId=272566&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-272566

**Comment by nk183** [comment]
@dkayiwa @gracepotma 
please merge this PR ,if any error please let me know

**Comment by dkayiwa** [comment]
@nk183 when you log in jira, there is an option to attach files. Can you attach user interface screenshots with the effects of your changes?

**Comment by nk183** [comment]
@dkayiwa  I have attached screenshot but [here](https://issues.openmrs.org/browse/OCLOMRS-857)

**Comment by nk183** [comment]
@dkayiwa something left please let me know

**Comment by nk183** [comment]
@dkayiwa ,please merge this PR

**Comment by gracepotma** [comment]
Thank you so much @nk183 for your work on this ticket, it's such a relief to have this done! I've already found our QA environment more useable now that I don't have to remember that the "back" button is hidden all the time.

**Pull request merged**